jobs/release.Jenkinsfile

```diff
@@ -96,11 +96,11 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
             """)
         }
 
-        def basearches = ""
-        if (params.ARCHES == '') {
-            basearches = shwrapCapture("jq -r '.builds | map(select(.id == \"${params.VERSION}\"))[].arches[]' builds/builds.json").split()
-        } else {
-            basearches = params.ARCHES.split()
+        def basearches = params.ARCHES.split()
+        def builtarches = shwrapCapture("jq -r '.builds | map(select(.id == \"${params.VERSION}\"))[].arches[]' builds/builds.json").split()
+        if builtarches != basearches {
+            echo "Some requested architectures did not successfully build!"
```

Comment:
Hmm, I think we should error out here instead if `basearches` isn't a subset of `builtarches`.

But the issue then is that we won't release at all if aarch64 failed but x86_64 didn't. Which is why the special empty string value for `ARCHES` was added.

Reply:
Ehh. I think maybe we should make "special empty value" not so special and be more explicit. I added a second commit to add a `ALLOW_MISSING_ARCHES` parameter.